src/Account/Entities/InvoiceQuery.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace UKFast\SDK\Account\Entities;
+
+class InvoiceQuery
+{
+    /**
+     * @var int
+     */
+    public $id;
+
+    /**
+     * @var int
+     */
+    public $contact_id;
+
+    /**
+     * @var int
+     */
+    public $amount;
+
+    /**
+     * @var string
+     */
+    public $what_was_expected;
```

Comment:
You're defining these in snake_case, yet the constructor defines them as camelCase?